modules/monitoring-knative-autoscaler-pods.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * serverless/monitoring-serverless.adoc
+
+[id="monitoring-knative-autoscaler-pods_{context}"]
+== Monitoring Knative Serving autoscaler pods using the {product-title} monitoring stack
+
+This section provides example  instructions for the visualization of Knative Serving pod autoscaling metrics by using the {product-title} monitoring tools.
+
+.Prerequisites
+
+* You must have the {product-title} monitoring stack installed. See the documentation about https://docs.openshift.com/container-platform/4.1/monitoring/cluster-monitoring/configuring-the-monitoring-stack.html#configuring-the-monitoring-stack[Configuring the monitoring stack].
```

Comment:
A couple of notes about this link:

1. We don't want to include specific versions, as these will need to be maintained moving forward. Since this is a reference to another entry in our doc set, it should be an xref. By using xrefs we keep the version and link updated moving forward.
2. As a continuation to the above, our current policies indicate that xref's should not be used in modules. The advice given thus far is to remove the link, keep the prerequisite as is, and include the specific module into the assembly as needed. Note that this has been a bit of a sticking point among writers.
3. If the link is vital, then I would move this to the assembly level, where xrefs are allowed.